### PR TITLE
New Kannada Cheat sheet

### PR DESCRIPTION
**Pull request by pnpninja**
kannada cheatsheet

IA Page : https://duck.co/ia/view/kannada_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Kannada Cheatsheet](https://duck.co/ia/view/kannada_cheatsheet)

**Description**: A cheatsheet for Kannada. Useful for travellers (especially SURGE attendees)

**Example Query**: [kannada cheatsheet](https://beta.duckduckgo.com/?q=kannada cheatsheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: http://www.omniglot.com/language/phrases/kannada.php

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/kannada_cheatsheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/kannada_cheatsheet)*

File: share/goodie/cheat_sheets/json/language/kannada.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
+{
+    "id": "kannada_cheat_sheet_cheat_sheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/kannada.json]: This should be `kannada_cheat_sheet`.

File: share/goodie/cheat_sheets/json/language/kannada.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    "description": "Basic Kannada words and phrases",
+
+    "metadata": {
+        "sourceName": "OmniGlot",
+        "sourceUrl" : "http://www.omniglot.com/language/phrases/kannada.php"
+    },
+
+    "aliases": [
+       "kannada phrases",
+       "english to kannada",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/kannada.json]: Lose these two (the ones with `phrases`).

File: share/goodie/cheat_sheets/json/language/kannada.json [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+       "basic kannada phrases",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/kannada.json]: Lose the aliases with `phrases` in the name.

**Comment by pnpninja** [comment]
All done !

**Comment by pnpninja** [comment]
@GuiltyDolphin : when might it go to beta ?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

**Comment by pnpninja** [comment]
Hmmm... I tested on Android native and Safari iOS. The symbols were rendered...

**Comment by GuiltyDolphin** [comment]
@pnpninja Don't worry - its a known bug apparently. It's showing up fine for me on Beta.

File: share/goodie/cheat_sheets/json/language/kannada.json [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+            "val" : "നನೇರವಾಗಿ ಮುಂದಕ್ಕೆ ಹೋಗಿ.",
+            "trn" : "nēravāgi mundakke hōgi."
+         },
+         {
+            "key" : "Where is the exit?",
+            "val" : "ನಿರ್ಗಮನ ಎಲ್ಲಿದೆ?",
+            "trn" : "nirgamana ellide?"
+         },
+         {
+            "key" : "Where?",
```
Comment (pnpninja) [inline share/goodie/cheat_sheets/json/language/kannada.json]: "Where" translation is wrong

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

**Comment by pnpninja** [comment]
@GuiltyDolphin all bugs fixed. Tested on Safari (iOS OSX), Android Native, Microsoft edge, firefox mac n windows

**Comment by GuiltyDolphin** [comment]
@pnpninja The ID of the IA page needs to be updated to be `kannada_cheat_sheet`.

/cc @moollaza

**Comment by pnpninja** [comment]
I can't seem to edit the ID of IA page.. Do I just change it here on the son file instead?

**Comment by GuiltyDolphin** [comment]
@pnpninja No, we can't edit it - I've cc'd @moollaza and hopefully he can update it.

**Comment by pnpninja** [comment]
@GuiltyDolphin Awesome.

**Comment by pnpninja** [comment]
@GuiltyDolphin I think its just easier that I edit the ID in my JSON file.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

**Comment by pnpninja** [comment]
@GuiltyDolphin I've changed the translation for 'where'. But the old translation is still popping up in beta.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/kannada.json [hunk 5]
```diff
@@ -0,0 +1,249 @@
+{
+    "id": "kannada_cheatsheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/kannada.json]: The ID **must** be `kannada_cheat_sheet`. The IA page will get updated at some point.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=kannada cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/kannada_cheatsheet_mobile.png)

**Comment by pnpninja** [comment]
The "where" translation has been corrected even on beta.. I guess it can be made live

**Comment by pnpninja** [comment]
ping @moollaza

**Comment by pnpninja** [comment]
ping @moollaza. This is ready. Have a look

**Comment by moollaza** [comment]
>@GuiltyDolphin was telling that you could edit the ID of the IA page ( I didn't follow the right format). Could you correct it if possible?

Yup :+1:  it's done. I've updated the link in the PR description

**Comment by moollaza** [comment]
@pnpninja are you a native speaker? If not I'd like to ask the community to review the translations to confirm they're OK.

**Comment by pnpninja** [comment]
@moollaza Its my mother tongue..

**Comment by moollaza** [comment]
@pnpninja Thanks! This LGTM :+1:

**Pull request merged**